java/ql/src/Telemetry/ExternalLibraryUsage.ql

```diff
@@ -0,0 +1,15 @@
+/**
+ * @name External libraries
+ * @description A list of external libraries used in the code
+ * @id java/telemetry/external-libs
+ */
+
+import java
+import ExternalAPI
+
+from ExternalAPI api
+where not api.getDeclaringType() instanceof TestLibrary
+// TODO [bm]: the count is not aggregated and we have the same jar with multiple usages, e.g.
+// 1	protobuf-java-3.17.3.jar	373
+// 2	protobuf-java-3.17.3.jar	48
```

Comment:
That's because you're aggregating by `ExternalAPI api` aka `Callable` rather than the jar. If you include `api` in your select clause then it should become a bit more clear what's going on.

Reply:
Yes, thanks for the input, solved locally, going to push in a minute.